src/utils.ts

```diff
@@ -328,22 +328,28 @@ export function escapeRegExp(string: string): string {
 }
 
 export function globToRegexp(glob: string, extended?: any): string {
```

Comment:
Any particular reason why `extended` is not restricted to be a boolean?

Reply:
Not sure - I guess whoever converted it to typescript just didn't manage to work out the correct type.